java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -88,7 +88,7 @@ else if (osName.toLowerCase().contains("linux")) {
             if (version == null) {
                 throw new UnsupportedOperationException("Cannot determine linux version");
             }
-            return distro.trim().replaceAll(" ", "_") + "." + version;
+            return (distro.trim().replaceAll(" ", "_") + "_" + version).toLowerCase();
```

Comment:
Lowercasing should be fine, but we have to update `build.sh` then.  It should be straightforward to do, just update lines 19, 24, 29, 49, 56, 61, and 171.  The last line might be a little tricky.  I really think adding the minor version to the name is problematic.